_includes/head.html

```diff
@@ -23,13 +23,17 @@
   <!-- i need to investigate on @font-face and render blocking resorces (css) -->
   <!--<link rel="preload" href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,700|Arvo:400,700' type='text/css' as="style" crossorigin="anonymous">-->
   <link rel="preload" as="script" href="https://cdn.ampproject.org/v0.js">
-  <link rel="preload" as="script" href="https://cdn.ampproject.org/v0/amp-experiment-0.1.js">
+  <!--<link rel="preload" as="script" href="https://cdn.ampproject.org/v0/amp-experiment-0.1.js">-->
```

Comment:
not sure if it is needed or deleting it will cause some issues, i'm testing it right now...